bpf/Makefile

```diff
@@ -120,15 +120,18 @@ CFLAGS_bpf_generic_lsm_output.o = -D__LARGE_BPF_PROG
 # Rules
 MTARGET_o  = $(patsubst $(DEPSDIR)%.d,$(OBJSDIR)%.o,$@)
 MTARGET_ll = $(patsubst $(DEPSDIR)%.d,$(OBJSDIR)%.ll,$@)
+MTARGET_i = $(patsubst $(DEPSDIR)%.d,$(OBJSDIR)%.i,$@)
 
 rule_d_CFLAGS  = $(CFLAGS_$(notdir $(MTARGET_o)))
 rule_o_CFLAGS  = $(CFLAGS_$(notdir $@))
 rule_ll_CFLAGS = $(CFLAGS_$(notdir $(patsubst %.ll,%.o,$@)))
 
 rule_d  = $(CLANG) $(CLANG_FLAGS) $(rule_d_CFLAGS) $1 -MM -MP -MT $(MTARGET_o)  $< >  $@ && \
-          $(CLANG) $(CLANG_FLAGS) $(rule_d_CFLAGS) $1 -MM -MP -MT $(MTARGET_ll) $< >> $@
+          $(CLANG) $(CLANG_FLAGS) $(rule_d_CFLAGS) $1 -MM -MP -MT $(MTARGET_ll) $< >> $@ && \
+          $(CLANG) $(CLANG_FLAGS) $(rule_d_CFLAGS) $1 -MM -MP -MT $(MTARGET_i) $< >> $@
```

Comment:
I wasn't sure about appending .ll rules to deps file.. and now we have one more ;-) but I think it's fine, it should not grow further I guess

Reply:
The other option would be to have explicit rules for .i objects, and I'm not sure how to do that if we want to generate them with the appropriate CFLAGS (e.g., -Dpizzaisthebest).